open-metadata-implementation/access-services/analytics-modeling/analytics-modeling-api/src/main/java/org/odpi/openmetadata/accessservices/analyticsmodeling/api/AnalyticsModelingSynchronization.java

```diff
@@ -4,36 +4,47 @@
 
 package org.odpi.openmetadata.accessservices.analyticsmodeling.api;
 
-import org.odpi.openmetadata.accessservices.analyticsmodeling.responses.AnalyticsModelingOMASAPIResponse;
+import org.odpi.openmetadata.accessservices.analyticsmodeling.ffdc.exceptions.AnalyticsModelingCheckedException;
+import org.odpi.openmetadata.accessservices.analyticsmodeling.model.ResponseContainerAssets;
 import org.odpi.openmetadata.accessservices.analyticsmodeling.synchronization.model.AnalyticsAsset;
+import org.odpi.openmetadata.frameworks.connectors.ffdc.PropertyServerException;
 
 public interface AnalyticsModelingSynchronization {
 
 	/**
-	 * Create analytics artifact defined as json input.
+	 * Create analytics artifact defined as input.
 	 * @param userId requested the operation.
 	 * @param serverCapability source where artifact persist.
```

Comment:
It is unique string. As example I use url, but it can be meaningful name of the 3rd party storage where artifacts are stored originally. The only restriction is it should be unique for each storage.